sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -5,7 +5,16 @@
 ### Breaking Changes
 
 - Removed `Azure::Storage::Blobs::PackageVersion`.
-- Renamed `GetUserDelegationKeyOptions::startsOn` to `GetUserDelegationKeyOptions::StartsOn`.
+- Renamed `GetUserDelegationKeyOptions::startsOn` to `StartsOn`.
+- Replaced all paginated collection functions that have the SinglePage suffix with pageable functions returning a `PagedResponse<T>`-derived type. The options are also renamed accordingly.
+  - `BlobServiceClient::ListBlobContainers()`.
+  - `BlobServiceClient::FindBlobsByTags()`.
+  - `BlobContainerClinet::ListBlobs()`.
+  - `BlobContainerClient::ListBlobsByHierarchy()`.
+  - `PageBlobClient::GetPageRanges()`.
+  - `PageBlobClient::GetPageRangesDiff()`.
+  - `PageBlobClient::GetManagedDiskPageRangesDiff()`.
+- Renamed `FilterBlobItem` to `TaggedBlobItem`.
```

Comment:
Maybe add an example of how a method return type got renamed, especially since we renamed some of the public fields too.
```suggestion
- Renamed `FilterBlobItem` to `TaggedBlobItem`.
  - FindBlobsByTagsSinglePageResult::Items renamed to FindBlobsByTagsPagedResponse::TaggedBlobs
```